docs/react-three-fiber/advanced/pitfalls.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Performance pitfalls'
+description: Some very important things to know for smooth results
+nav: 3
+---
+
+## WebGL performance pitfalls
+
+### Tips and Tricks <a id="tips-and-tricks"></a>
+
+This is the best overview I could find: https://discoverthreejs.com/tips-and-tricks
+
+The most important is gotcha in Threejs is that creating objects can be expensive, think twice before you mount/unmnount things! Every material or light that you put into the scene has to compile, every geometry you create will be processed. Share materials and geometries if you can, either in global scope or locally:
```

Comment:
typo here "the most important is gotcha"

Reply:
Fixed